docs/src/Datapoint.doc.js

```diff
@@ -0,0 +1,140 @@
+// @flow strict
+import React, { type Node } from 'react';
+import { Datapoint } from 'gestalt';
+import PropTable from './components/PropTable.js';
+import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
+import FeedbackCallout from './components/FeedbackCallout.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(<FeedbackCallout componentName="Datapoint" />);
+
+card(
+  <PageHeader
+    name="Datapoint"
+    description="Datapoint displays at-a-glance data for a user to quickly view key metrics."
+    defaultCode={`
+<Datapoint size="lg" infoText="Contextual information for this Datapoint" title="Datapoint title" value="1.23M" trend={{value: 30, accessibilityLabel: "Trending up"}} helperText="Datapoint helper text" />
+`}
+    pilot
+  />,
+);
+
+// Specifying Component gives auto warnings about missing props
+card(
+  <PropTable
+    Component={Datapoint}
+    props={[
+      {
+        name: 'title',
+        type: 'string',
+        description: 'The header for the component',
+        required: true,
+      },
+      {
+        name: 'value',
+        type: 'string',
+        description: 'The main datapoint value (e.g., 1.23M)',
+        required: true,
+      },
+      {
+        name: 'trend',
+        type: '{| accessibilityLabel: string, value: string |}',
+        description: `Object detailing the trend value (change in time - e.g., +30%), and accessibilityLabel to describe the trend's icon (e.g., "Trending up")`,
+      },
+      {
+        name: 'trendSignal',
+        type: `'good' | 'bad' | 'neutral' | 'auto'`,
+        defaultValue: 'auto',
+        description: `A visual indicator whether the trend is considered "good", "bad" or "neutral". By setting \`trendSignal\` to \`auto\`, a positive trend will be considered "good", a negative trend will be considered "bad" and a trend of zero will be considered "neutral"`,
+      },
+      {
+        name: 'infoText',
```

Comment:
`tooltipText`?

Reply:
My only concern with that is binding the name to a treatment. What if we change the name of tooltip? What if we use a different form of UI to display at some point?

Reply:
I think given how we do this type of prop in other places (for example, badgeText), it would make sense to air on the side of clarity with `tooltipText`. If we change the name of Tooltip, it's an easy codemod to change this prop name, and if we use a different treatment, it'd be a breaking change anyway, so we'd probably have more then just the prop name that would need updating. Keeping it specific now allows to expand in the future without worrying about name collisions (for example, if we wanted to add `helperText` that is always visible below the title)

Reply:
OK, sounds good. I'll switch it over to tooltipText. Thanks folks!